src/main/java/org/aszjch/demoapp/infrastructure/storage/ExternalStorageService.java

```diff
@@ -0,0 +1,9 @@
+package org.aszjch.demoapp.infrastructure.storage;
+
+import io.minio.ObjectWriteResponse;
+import org.springframework.web.multipart.MultipartFile;
+
+public interface ExternalStorageService {
```

Comment:
Kurde, nie podoba mi się serwajs wszędzie. Spróbujmy to lepiej ponazywać. Wydaje mi się, że tutaj ExternalStorage wystarczy, a klasa implementująca to np. MinIOStorage.

Reply:
Może ExternalStorageClient? Jak ktoś jest upośledzony jak ja, to widząc ExternalStorage pomyśli, że to już jest ten storage i pliki znajdują się wewnątrz klasy, a nie, że jest to klient służący do komunikacji z tym storage'm